src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue

```diff
@@ -144,11 +166,40 @@
             <p>You need to provide the following items.</p>
           </v-card-text>
         </v-card>
-        <ApplicationSummaryTranscriptListItem
-          v-for="transcript in waitingForDetailsTranscripts"
-          :key="transcript.id?.toString()"
-          :name="transcript.educationalInstitutionName"
-          :status="transcript.status"
+        <template v-for="transcript in waitingForDetailsTranscripts" :key="transcript.id?.toString()">
+          <ApplicationSummaryHeader :text="getTranscriptHeaderString(transcript)" />
+          <ApplicationSummaryActionListItem
+            :active="!transcript.transcriptReceivedByRegistry"
+            :text="`Transcript: ${transcript.educationalInstitutionName}`"
+            :go-to="() => router.push({ name: 'manageTranscript', params: { applicationId: route.params.applicationId } })"
+          />
+          <ApplicationSummaryActionListItem
+            v-if="transcript.educationRecognition === 'NotRecognized'"
+            :active="!transcript.courseOutlineReceivedByRegistry"
+            :text="`Course outlines or syllabi: ${transcript.educationalInstitutionName}`"
+            :go-to="() => router.push({ name: 'manageCourseOutline', params: { applicationId: route.params.applicationId } })"
+          />
+          <ApplicationSummaryActionListItem
+            v-if="transcript.educationRecognition === 'NotRecognized' && !applicationStatus?.certificationTypes?.includes('EceAssistant')"
+            :active="!transcript.programConfirmationReceivedByRegistry"
+            :text="`Program Confirmation Form: ${transcript.educationalInstitutionName}`"
+            :go-to="() => router.push({ name: 'manageTranscript', params: { applicationId: route.params.applicationId } })"
+          />
+          <ApplicationSummaryActionListItem
+            v-if="transcript.educationRecognition === 'NotRecognized' && transcript.country?.countryName?.toLowerCase() !== 'canada'"
```

Comment:
should we use the method in the configStore that does this? Config.ts

canada(state) {
      return state.countryList.find((country) => country.countryName!.toLowerCase() === "canada");
    },

Reply:
Hmmm... I looked at this but wasn't sure about getting the whole object, we probably need to add `toLowerCase()` to wherever we are using this function or create a helper `lowerCaseCanada`? For example here we aren't ensuring these strings match if we have case mismatch:

```
    isCanada(): boolean {
      return this.modelValue.country === this.configStore.canada?.countryName;
    },
  ```